benchmarks/benchmark_latency.py

```diff
@@ -211,5 +212,11 @@ def run_to_completion(profile_dir: Optional[str] = None):
         type=str,
         default=None,
         help='Path to save the latency results in JSON format.')
+    parser.add_argument('--gpu-memory-utilization',
```

Comment:
Wondering if we want to expose max-model-len as well in the args. We are doing that in the benchmark_througput ( https://sourcegraph.com/github.com/vllm-project/vllm/-/blob/benchmarks/benchmark_throughput.py?L301) and wondering if we should add it here as well since that is another parameter to tune for this error?